src/commands/consolidate.py

```diff
@@ -0,0 +1,330 @@
+    envvar='LOG_LEVEL',
+    help='The log level.',
+)
+@click.command(help='Performs a vault validators consolidation after pectra upgrade.')
+# pylint: disable-next=too-many-arguments,too-many-locals
+def consolidate(
+    network: str,
+    vault: ChecksumAddress,
+    execution_endpoints: str,
+    consensus_endpoints: str,
+    data_dir: str,
+    hot_wallet_file: str | None,
+    hot_wallet_password_file: str | None,
+    verbose: bool,
+    no_confirm: bool,
+    log_level: str,
+    count: int | None = None,
+    from_keys: list[HexStr] | None = None,
+    to_key: HexStr | None = None,
+) -> None:
+    operator_config = OperatorConfig(Path(data_dir))
+    operator_config.load(network=network)
+
+    settings.set(
+        vaults=[vault],
+        network=operator_config.network,
+        data_dir=operator_config.data_dir,
+        execution_endpoints=execution_endpoints,
+        consensus_endpoints=consensus_endpoints,
+        hot_wallet_file=hot_wallet_file,
+        hot_wallet_password_file=hot_wallet_password_file,
+        verbose=verbose,
+        log_level=log_level,
+    )
+    try:
+        asyncio.run(
+            main(
+                vault_address=vault,
+                count=count,
+                from_keys=from_keys,
+                to_key=to_key,
+                no_confirm=no_confirm,
+            )
+        )
+    except Exception as e:
+        log_verbose(e)
+        sys.exit(1)
+
+
+async def main(
+    vault_address: ChecksumAddress,
+    count: int | None,
+    from_keys: list[HexStr] | None,
+    to_key: HexStr | None,
+    no_confirm: bool,
+) -> None:
+    setup_logging()
+    await setup_clients()
+    await _check_validators_manager()
+    chain_head = await get_chain_finalized_head()
+
+    if from_keys is not None and to_key is not None:
+        target_source_public_keys = await _get_selected_target_source_public_keys(
+            from_keys=from_keys,
+            to_key=to_key,
+        )
+
+    else:
+        target_source_public_keys = await _get_all_target_source_public_keys(
+            vault_address=vault_address,
+            block_number=chain_head.block_number,
+            count=count,
+        )
+
+    click.secho(
+        f'Consolidating {len(target_source_public_keys)} validators: ',
+    )
+    for target_key, source_key in target_source_public_keys:
+        click.secho(f'    {source_key} -> {target_key}')
+    if not no_confirm:
+        click.confirm(
+            'Proceed consolidation?',
+            default=True,
+            abort=True,
+        )
+    current_fee = await get_request_fee(
+        settings.network_config.CONSOLIDATION_CONTRACT_ADDRESS, block_number=chain_head.block_number
+    )
+    if current_fee > MAX_CONSOLIDATION_REQUEST_FEE:
+        logger.info(
+            'Consolidation is skipped because high consolidation fee, current fees is %s. '
+            'Increase MAX_CONSOLIDATION_REQUEST_FEE env var to allow it',
+            current_fee,
+        )
```

Comment:
Suggest to rephrase: "The current consolidation fee (%s Gwei) exceeds the maximum allowed (%s Gwei). You can override the limit using the MAX_CONSOLIDATION_REQUEST_FEE environment variable."